### ISSUE TITLE
Different prompt formats for different models

### ISSUE THREAD

**Issue by jeremysalwen** [Pelochus/ezrknn-llm#11]
FYI something I noticed is that the rkllm code is using the wrong prompt format for some models.  For Phi3, for example, it uses different prompt tags.  Probably it should give different prompts for different models, or at least give a warning about it.

**Comment by Pelochus** [comment]
Yes I'm aware of this issue. I will try to find a fix whenever I have some time (been low on free time for a while... sorry)
